plugins/generator-datapack-1.16.x/datapack-1.16.x/templates/pack.mcmeta.ftl

```diff
@@ -1,6 +1,6 @@
 {
     "pack": {
-        "pack_format": 4,
+        "pack_format": 5,

```

Comment:
Same here, but I'm not really sure about this one
It's 5 before 1.16.2 and 6 since 1.16.2 but generator is for 1.16.x

Reply:
I will use 6 as we target the latest releases